File: Interview-150/extra/TwoSum.py
class Solution:
    def twoSum(self, nums :List[int], target:int)-> bool:
        seen={}
        for i, num in enumerate(nums):
            res= target-nums[i]
            if(res in seen):
                return [seen[res],i]
            seen[num]=i
        return[]

File: Interview-150/extra/TwoSum.java
import java.util.*;
class TwoSum{
    public int[] twoSum(int[]nums,int target){
        Map<Integer,Integer> res= new HashMap<>();
        for(int i=0;i<nums.length;i++){
            int s= target-nums[i];
            if(res.containsKey(s)){
                return new int[]{res.get(s),i};
            }
            res.put(nums[i],i);
        }
        return new int[]{};
    }
}